subsys/mgmt/Kconfig.mcumgr

```diff
@@ -53,13 +58,33 @@ menuconfig MCUMGR_CMD_IMG_MGMT
 	  Enables mcumgr handlers for image management
 
 if MCUMGR_CMD_IMG_MGMT
-	config IMG_MGMT_UL_CHUNK_SIZE
+config IMG_MGMT_UL_CHUNK_SIZE
 	int "Maximum chunk size for image uploads"
 	default 512
 	help
 	  Limits the maximum chunk size for image uploads, in bytes.  A buffer of
 	  this size gets allocated on the stack during handling of a image upload
 	  command.
+
+config IMG_MGMT_LAZY_ERASE
```

Comment:
is this similar to [IMG_ERASE_PROGRESSIVELY](https://github.com/zephyrproject-rtos/zephyr/blob/master/subsys/dfu/Kconfig#L53)

Reply:
check that it is similar. looks like not easy to use flash_util  in mcumgr now

Reply:
It sure is, sorry for not stating it promptly. I will refactor in order to set IMG_MGMT_LAZY_ERASE with IMG_ERASE_PROGRESSIVELY.